common/decisions/dvg_atlantia.txt

```diff
@@ -0,0 +1,131 @@
+dvg_atlantia_referendum = {
+	is_shown = {
+
+		this = c:ATL
+		NOT = { has_global_variable = atlantia_independent }
+		
+	}
+
+    possible = {
+		year >= 1840
+	
+	}
+
+    when_taken = {
+		trigger_event = { id = dvg_atlantia.1 popup = yes }
+    }
+
+	ai_chance = {
+		base = 100
+	}
+}
+
+dvg_nouvellsatian_support = {
+	is_shown = {
+
+		this = c:ATL
+		
+		NOT = { exists = global_var:nouvellsatian_support }
+		
+	}
+
+    possible = {
+		
+		any_neighbouring_state = {
+			owner = c:NEA

```

Comment:
Check if c:NEA exists, its now giving error. Maybe we should not show the decision if the NEA country tag does not exist. As it also make it look weird in the UI. This is an issue with all decisions with NEA when it does not exist.
![image](https://user-images.githubusercontent.com/116951759/227636854-83b2c2e5-5b11-4223-b3a0-afc1f0293261.png)